### PR TITLE
New CTDMO cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Autogenerated during QCTs.  I already did a thorough review before posting these, so they just need one additional review.

### CODE REVIEW

File: calibration/CTDMOG/CGINS-CTDMOG-13433__20220123.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+37-13433,CC_ptca0,5.245724e+05,
+37-13433,CC_ptca1,2.041517e+01,
+37-13433,CC_ptca2,-4.109457e-01,
+37-13433,CC_ptcb0,2.500429e+01,
+37-13433,CC_ptcb1,4.488778e-04,
+37-13433,CC_ptcb2,0.000000e+00,
+37-13433,CC_ptempa0,1.530229e+02,
+37-13433,CC_ptempa1,-7.943806e-02,
+37-13433,CC_ptempa2,3.578528e-06,
+37-13433,CC_wbotc,3.101952e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13433__20220123.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13434__20220125.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+37-13434,CC_ptca0,5.252700e+05,
+37-13434,CC_ptca1,1.133336e+01,
+37-13434,CC_ptca2,-2.940194e-01,
+37-13434,CC_ptcb0,2.508950e+01,
+37-13434,CC_ptcb1,-3.000000e-04,
+37-13434,CC_ptcb2,0.000000e+00,
+37-13434,CC_ptempa0,1.460965e+02,
+37-13434,CC_ptempa1,-6.417550e-02,
+37-13434,CC_ptempa2,-2.371211e-07,
+37-13434,CC_wbotc,2.765135e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13434__20220125.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13435__20220125.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+37-13435,CC_ptca0,5.242156e+05,
+37-13435,CC_ptca1,4.092828e+00,
+37-13435,CC_ptca2,5.682414e-02,
+37-13435,CC_ptcb0,2.523463e+01,
+37-13435,CC_ptcb1,-2.750000e-04,
+37-13435,CC_ptcb2,0.000000e+00,
+37-13435,CC_ptempa0,-9.184816e+01,
+37-13435,CC_ptempa1,4.622314e-02,
+37-13435,CC_ptempa2,-3.718930e-07,
+37-13435,CC_wbotc,3.715942e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13435__20220125.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13436__20220211.csv [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+37-13436,CC_ptca0,5.249764e+05,
+37-13436,CC_ptca1,-4.928204e-01,
+37-13436,CC_ptca2,7.242029e-02,
+37-13436,CC_ptcb0,2.545362e+01,
+37-13436,CC_ptcb1,1.250000e-04,
+37-13436,CC_ptcb2,0.000000e+00,
+37-13436,CC_ptempa0,1.585149e+02,
+37-13436,CC_ptempa1,-7.420086e-02,
+37-13436,CC_ptempa2,2.219372e-06,
+37-13436,CC_wbotc,3.426067e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13436__20220211.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13437__20220209.csv [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+37-13437,CC_ptca0,5.239063e+05,
+37-13437,CC_ptca1,2.146400e+00,
+37-13437,CC_ptca2,-3.284871e-02,
+37-13437,CC_ptcb0,2.514510e+01,
+37-13437,CC_ptcb1,-7.250000e-04,
+37-13437,CC_ptcb2,0.000000e+00,
+37-13437,CC_ptempa0,-5.896476e+01,
+37-13437,CC_ptempa1,5.561897e-02,
+37-13437,CC_ptempa2,-9.091839e-07,
+37-13437,CC_wbotc,2.464384e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13437__20220209.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13438__20220205.csv [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+37-13438,CC_ptca0,5.250217e+05,
+37-13438,CC_ptca1,-1.459533e+00,
+37-13438,CC_ptca2,1.704591e-01,
+37-13438,CC_ptcb0,2.512775e+01,
+37-13438,CC_ptcb1,1.500000e-04,
+37-13438,CC_ptcb2,0.000000e+00,
+37-13438,CC_ptempa0,1.352562e+02,
+37-13438,CC_ptempa1,-5.785518e-02,
+37-13438,CC_ptempa2,-2.357904e-06,
+37-13438,CC_wbotc,3.040949e-08,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13438__20220205.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13439__20220125.csv [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+37-13439,CC_ptca0,5.253803e+05,
+37-13439,CC_ptca1,4.180443e+00,
+37-13439,CC_ptca2,-1.130422e-01,
+37-13439,CC_ptcb0,2.498400e+01,
+37-13439,CC_ptcb1,2.000000e-04,
+37-13439,CC_ptcb2,0.000000e+00,
+37-13439,CC_ptempa0,1.829687e+02,
+37-13439,CC_ptempa1,-7.235992e-02,
+37-13439,CC_ptempa2,1.219046e-06,
+37-13439,CC_wbotc,2.180997e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13439__20220125.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13593__20220125.csv [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+37-13593,CC_ptca0,5.251400e+05,
+37-13593,CC_ptca1,-4.581214e+00,
+37-13593,CC_ptca2,3.307869e-01,
+37-13593,CC_ptcb0,2.520712e+01,
+37-13593,CC_ptcb1,2.250000e-04,
+37-13593,CC_ptcb2,0.000000e+00,
+37-13593,CC_ptempa0,1.534941e+02,
+37-13593,CC_ptempa1,-7.076636e-02,
+37-13593,CC_ptempa2,1.109581e-06,
+37-13593,CC_wbotc,6.487438e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13593__20220125.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13603__20220121.csv [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+37-13603,CC_ptca0,5.259018e+05,
+37-13603,CC_ptca1,-1.534206e+00,
+37-13603,CC_ptca2,2.452063e-01,
+37-13603,CC_ptcb0,2.505638e+01,
+37-13603,CC_ptcb1,-1.246883e-04,
+37-13603,CC_ptcb2,0.000000e+00,
+37-13603,CC_ptempa0,1.581634e+02,
+37-13603,CC_ptempa1,-7.254933e-02,
+37-13603,CC_ptempa2,1.026931e-06,
+37-13603,CC_wbotc,1.859974e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13603__20220121.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13604__20220123.csv [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+37-13604,CC_ptca0,5.251369e+05,
+37-13604,CC_ptca1,-2.053032e+00,
+37-13604,CC_ptca2,3.720309e-02,
+37-13604,CC_ptcb0,2.522763e+01,
+37-13604,CC_ptcb1,-2.750000e-04,
+37-13604,CC_ptcb2,0.000000e+00,
+37-13604,CC_ptempa0,1.655913e+02,
+37-13604,CC_ptempa1,-7.274692e-02,
+37-13604,CC_ptempa2,4.383309e-07,
+37-13604,CC_wbotc,4.644981e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13604__20220123.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13605__20220125.csv [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+37-13605,CC_ptca0,5.251439e+05,
+37-13605,CC_ptca1,-2.757753e+00,
+37-13605,CC_ptca2,1.184773e-01,
+37-13605,CC_ptcb0,2.530837e+01,
+37-13605,CC_ptcb1,6.750001e-04,
+37-13605,CC_ptcb2,0.000000e+00,
+37-13605,CC_ptempa0,1.621445e+02,
+37-13605,CC_ptempa1,-5.917588e-02,
+37-13605,CC_ptempa2,-1.595223e-06,
+37-13605,CC_wbotc,6.012297e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13605__20220125.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13606__20220125.csv [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+37-13606,CC_ptca0,5.237135e+05,
+37-13606,CC_ptca1,-8.093879e-02,
+37-13606,CC_ptca2,4.219728e-02,
+37-13606,CC_ptcb0,2.504471e+01,
+37-13606,CC_ptcb1,-7.518798e-05,
+37-13606,CC_ptcb2,0.000000e+00,
+37-13606,CC_ptempa0,-6.453108e+01,
+37-13606,CC_ptempa1,5.197715e-02,
+37-13606,CC_ptempa2,-5.257605e-07,
+37-13606,CC_wbotc,-9.588134e-08,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13606__20220125.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13607__20220123.csv [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+37-13607,CC_ptca0,5.245993e+05,
+37-13607,CC_ptca1,-3.840251e+00,
+37-13607,CC_ptca2,7.338820e-02,
+37-13607,CC_ptcb0,2.502675e+01,
+37-13607,CC_ptcb1,-2.500000e-04,
+37-13607,CC_ptcb2,0.000000e+00,
+37-13607,CC_ptempa0,1.556587e+02,
+37-13607,CC_ptempa1,-6.314961e-02,
+37-13607,CC_ptempa2,-8.689643e-07,
+37-13607,CC_wbotc,3.126560e-08,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13607__20220123.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13608__20220125.csv [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+37-13608,CC_ptca0,5.243137e+05,
+37-13608,CC_ptca1,7.655278e+00,
+37-13608,CC_ptca2,6.952589e-02,
+37-13608,CC_ptcb0,2.533263e+01,
+37-13608,CC_ptcb1,1.250000e-04,
+37-13608,CC_ptcb2,0.000000e+00,
+37-13608,CC_ptempa0,1.688629e+02,
+37-13608,CC_ptempa1,-7.339069e-02,
+37-13608,CC_ptempa2,1.304384e-06,
+37-13608,CC_wbotc,4.465138e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13608__20220125.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13616__20220123.csv [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+37-13616,CC_ptca0,5.225398e+05,
+37-13616,CC_ptca1,2.884013e+00,
+37-13616,CC_ptca2,-2.075642e-01,
+37-13616,CC_ptcb0,2.527587e+01,
+37-13616,CC_ptcb1,-2.250000e-04,
+37-13616,CC_ptcb2,0.000000e+00,
+37-13616,CC_ptempa0,1.531998e+02,
+37-13616,CC_ptempa1,-6.410759e-02,
+37-13616,CC_ptempa2,-8.257323e-07,
+37-13616,CC_wbotc,3.471849e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13616__20220123.csv:25]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13620__20220125.csv [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+37-13620,CC_ptca0,5.247535e+05,
+37-13620,CC_ptca1,6.816670e-01,
+37-13620,CC_ptca2,1.949735e-02,
+37-13620,CC_ptcb0,2.512687e+01,
+37-13620,CC_ptcb1,-2.500000e-05,
+37-13620,CC_ptcb2,0.000000e+00,
+37-13620,CC_ptempa0,1.397320e+02,
+37-13620,CC_ptempa1,-6.773484e-02,
+37-13620,CC_ptempa2,1.514275e-07,
+37-13620,CC_wbotc,1.919601e-07,
```
Comment (cdobs) [inline calibration/CTDMOG/CGINS-CTDMOG-13620__20220125.csv:25]: looks good

File: calibration/CTDMOH/CGINS-CTDMOH-12223__20220123.csv [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+37-12223,CC_ptca0,5.248845e+05,
+37-12223,CC_ptca1,5.063452e-01,
+37-12223,CC_ptca2,-6.377991e-02,
+37-12223,CC_ptcb0,2.528238e+01,
+37-12223,CC_ptcb1,-1.250000e-04,
+37-12223,CC_ptcb2,0.000000e+00,
+37-12223,CC_ptempa0,-6.753656e+01,
+37-12223,CC_ptempa1,5.157993e-02,
+37-12223,CC_ptempa2,-5.627726e-07,
+37-12223,CC_wbotc,3.477213e-07,
```
Comment (cdobs) [inline calibration/CTDMOH/CGINS-CTDMOH-12223__20220123.csv:25]: looks good

File: calibration/CTDMOH/CGINS-CTDMOH-12391__20220123.csv [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+37-12391,CC_ptca0,5.246608e+05,
+37-12391,CC_ptca1,3.647513e+00,
+37-12391,CC_ptca2,-7.800096e-02,
+37-12391,CC_ptcb0,2.525850e+01,
+37-12391,CC_ptcb1,9.000000e-04,
+37-12391,CC_ptcb2,0.000000e+00,
+37-12391,CC_ptempa0,-6.828683e+01,
+37-12391,CC_ptempa1,5.141522e-02,
+37-12391,CC_ptempa2,-5.594929e-07,
+37-12391,CC_wbotc,1.628666e-07,
```
Comment (cdobs) [inline calibration/CTDMOH/CGINS-CTDMOH-12391__20220123.csv:25]: looks good

File: calibration/CTDMOH/CGINS-CTDMOH-12392__20220123.csv [hunk 19]
```diff
@@ -0,0 +1,25 @@
...
+37-12392,CC_ptca0,5.248484e+05,
+37-12392,CC_ptca1,-3.755782e+00,
+37-12392,CC_ptca2,3.956077e-02,
+37-12392,CC_ptcb0,2.529325e+01,
+37-12392,CC_ptcb1,6.500000e-04,
+37-12392,CC_ptcb2,0.000000e+00,
+37-12392,CC_ptempa0,-7.011707e+01,
+37-12392,CC_ptempa1,5.125845e-02,
+37-12392,CC_ptempa2,-5.283391e-07,
+37-12392,CC_wbotc,2.291483e-07,
```
Comment (cdobs) [inline calibration/CTDMOH/CGINS-CTDMOH-12392__20220123.csv:25]: looks good

File: calibration/CTDMOH/CGINS-CTDMOH-12393__20220125.csv [hunk 20]
```diff
@@ -0,0 +1,25 @@
...
+37-12393,CC_ptca0,5.243234e+05,
+37-12393,CC_ptca1,6.096621e-01,
+37-12393,CC_ptca2,2.181762e-02,
+37-12393,CC_ptcb0,2.524537e+01,
+37-12393,CC_ptcb1,8.750001e-04,
+37-12393,CC_ptcb2,0.000000e+00,
+37-12393,CC_ptempa0,-6.668200e+01,
+37-12393,CC_ptempa1,5.304567e-02,
+37-12393,CC_ptempa2,-7.490985e-07,
+37-12393,CC_wbotc,1.988334e-07,
```
Comment (cdobs) [inline calibration/CTDMOH/CGINS-CTDMOH-12393__20220125.csv:25]: looks good

File: calibration/CTDMOH/CGINS-CTDMOH-13445__20220205.csv [hunk 21]
```diff
@@ -0,0 +1,25 @@
...
+37-13445,CC_ptca0,5.241047e+05,
+37-13445,CC_ptca1,1.415677e+00,
+37-13445,CC_ptca2,-4.368246e-02,
+37-13445,CC_ptcb0,2.493250e+01,
+37-13445,CC_ptcb1,-1.000000e-04,
+37-13445,CC_ptcb2,0.000000e+00,
+37-13445,CC_ptempa0,1.932789e+02,
+37-13445,CC_ptempa1,-1.216276e-01,
+37-13445,CC_ptempa2,8.779516e-06,
+37-13445,CC_wbotc,3.107259e-07,
```
Comment (cdobs) [inline calibration/CTDMOH/CGINS-CTDMOH-13445__20220205.csv:25]: looks good

File: calibration/CTDMOH/CGINS-CTDMOH-13446__20220123.csv [hunk 22]
```diff
@@ -0,0 +1,25 @@
...
+37-13446,CC_ptca0,5.241308e+05,
+37-13446,CC_ptca1,-2.285080e+00,
+37-13446,CC_ptca2,6.234971e-02,
+37-13446,CC_ptcb0,2.507650e+01,
+37-13446,CC_ptcb1,3.000000e-04,
+37-13446,CC_ptcb2,0.000000e+00,
+37-13446,CC_ptempa0,2.075245e+02,
+37-13446,CC_ptempa1,-9.295250e-02,
+37-13446,CC_ptempa2,1.995279e-08,
+37-13446,CC_wbotc,2.785668e-07,
```
Comment (cdobs) [inline calibration/CTDMOH/CGINS-CTDMOH-13446__20220123.csv:25]: looks good

File: calibration/CTDMOH/CGINS-CTDMOH-13450__20220123.csv [hunk 23]
```diff
@@ -0,0 +1,25 @@
...
+37-13450,CC_ptca0,5.247182e+05,
+37-13450,CC_ptca1,2.743697e-01,
+37-13450,CC_ptca2,-3.247444e-02,
+37-13450,CC_ptcb0,2.516100e+01,
+37-13450,CC_ptcb1,-2.000000e-04,
+37-13450,CC_ptcb2,0.000000e+00,
+37-13450,CC_ptempa0,2.078624e+02,
+37-13450,CC_ptempa1,-9.638315e-02,
+37-13450,CC_ptempa2,7.033839e-07,
+37-13450,CC_wbotc,2.997000e-08,
```
Comment (cdobs) [inline calibration/CTDMOH/CGINS-CTDMOH-13450__20220123.csv:25]: looks good

File: calibration/CTDMOH/CGINS-CTDMOH-13451__20220121.csv [hunk 24]
```diff
@@ -0,0 +1,25 @@
...
+37-13451,CC_ptca0,5.246119e+05,
+37-13451,CC_ptca1,-3.716180e+00,
+37-13451,CC_ptca2,4.365032e-02,
+37-13451,CC_ptcb0,2.528963e+01,
+37-13451,CC_ptcb1,-2.750000e-04,
+37-13451,CC_ptcb2,0.000000e+00,
+37-13451,CC_ptempa0,1.537845e+02,
+37-13451,CC_ptempa1,-9.028629e-02,
+37-13451,CC_ptempa2,-2.456798e-07,
+37-13451,CC_wbotc,2.477555e-07,
```
Comment (cdobs) [inline calibration/CTDMOH/CGINS-CTDMOH-13451__20220121.csv:25]: looks good

File: calibration/CTDMOH/CGINS-CTDMOH-13452__20220205.csv [hunk 25]
```diff
@@ -0,0 +1,25 @@
...
+37-13452,CC_ptca0,5.246141e+05,
+37-13452,CC_ptca1,2.784027e+00,
+37-13452,CC_ptca2,-6.944624e-02,
+37-13452,CC_ptcb0,2.508612e+01,
+37-13452,CC_ptcb1,-5.750000e-04,
+37-13452,CC_ptcb2,0.000000e+00,
+37-13452,CC_ptempa0,2.794387e+02,
+37-13452,CC_ptempa1,-1.333583e-01,
+37-13452,CC_ptempa2,7.040695e-06,
+37-13452,CC_wbotc,4.268153e-07,
```
Comment (cdobs) [inline calibration/CTDMOH/CGINS-CTDMOH-13452__20220205.csv:25]: looks good

File: calibration/CTDMOH/CGINS-CTDMOH-13650__20220125.csv [hunk 26]
```diff
@@ -0,0 +1,25 @@
...
+37-13650,CC_ptca0,5.237743e+05,
+37-13650,CC_ptca1,-1.568452e-01,
+37-13650,CC_ptca2,-2.358562e-02,
+37-13650,CC_ptcb0,2.510325e+01,
+37-13650,CC_ptcb1,5.000000e-05,
+37-13650,CC_ptcb2,0.000000e+00,
+37-13650,CC_ptempa0,1.608299e+02,
+37-13650,CC_ptempa1,-8.623406e-02,
+37-13650,CC_ptempa2,-3.588768e-07,
+37-13650,CC_wbotc,3.525529e-07,
```
Comment (cdobs) [inline calibration/CTDMOH/CGINS-CTDMOH-13650__20220125.csv:25]: looks good

**Review by cdobs — APPROVED** [review]
all look good

**Comment by rtravis-whoi** [comment]
[PYIV Instrument Cal CSVs [DT-158]](https://trello.com/c/I1InRCiO/158-pyiv-instrument-cal-csvs-dt-158)

**Pull request merged**